Samples/SessionReplay-CameraTest/SessionReplay-CameraTest.yml

```diff
@@ -5,7 +5,8 @@ projectReferences:
   Sentry:
     path: ../../Sentry.xcodeproj
 fileGroups:
-  - ./Config
+  - ../Shared/Config
+  - SessionReplay-CameraTest.yml
```

Comment:
`m`: Why does this PR change the camera test app here and not in #5106?

Reply:
most of these are a filesystem-level refactor to move the xcconfigs, that only lived inside the camera test app's directory, to a shared location.